src/code/z_player_lib.c

```diff
@@ -1482,8 +1509,45 @@ s32 Player_GetEnvironmentalHazard(PlayState* play) {
     return envHazard + 1;
 }
 
-void func_80124420(Player* player);
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_player_lib/func_80124420.s")
+void Player_UpdateBunnyEarsKinematics(Player* player) {
```

Comment:
Interesting that they moved this to player_lib from player. Worth noting this function in OoT dropped the `Kinematics` part and is called `Player_UpdateBunnyEars`. Since Bunny Ears have more functionality in MM, could `Player_UpdateBunnyEars` clash with something else?

Also, would be nice to copy the comments from OoT into this function here

Reply:
@EllipticEllipsis did this function in both OoT and MM, so I think it would be better if he answered this

Reply:
The additional MM functionality is just a check for current mask and a speed adjustment in a large Player function somewhere, I don't think it matters for this.

Reply:
Also I could have sworn I wrote a lot of documentation on this, what happened to it?

Reply:
So should we rename the function to match OoT?